### PR TITLE
app-text/unrtf: Fix call to undeclared function strcasestr

### PR DESCRIPTION
**Pull request by listout**
and update EAPI 7 -> 8

Closes: https://bugs.gentoo.org/894716

### CODE REVIEW

**Comment by Redfoxymoon** [comment]
Musl has `strcasestr` but it needs `_GNU_SOURCE` to be set, unrtf sets `__USE_GNU` instead, this change would result in different behavior

**Comment by listout** [comment]
@Redfoxymoon Thanks a lot for the review. I've added defining _GNU_SOURCE instead of using strstr. Does the patch look good or shall I remove the __USE_GNU definition?

**Comment by Redfoxymoon** [comment]
It shouldn't matter, only `_GNU_SOURCE` is needed to enable gnu extension functions

File: app-text/unrtf/files/unrtf-0.21.10-use-strstr.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Use strstr (standard) instead of strcasestr (non standard)
```
Comment (thesamesam) [inline app-text/unrtf/files/unrtf-0.21.10-use-strstr.patch:1]: The comment is stale but I'll fix that on merging now.

**Pull request closed without merging**